client/src/assets/data/filterOptions.js

```diff
@@ -17,15 +17,15 @@ export default {
     ],
   },
   type: {
-    label: "Type",
+    label: "type",
     className: "filter-3",
     options: [
-      "Medical supplies",
-      "Groceries / Food",
```

Comment:
Will the filter not work if you have the spaces between the / ? If they will, I think we should still have this for better presentation.

Reply:
@idjevm They need to match the backend `POST_TYPES` as defined. Interestingly the original design has spaces for "Groceries / Food" but no spaces for "Wellbeing/Mental". But I would say this it's fine the data model was approved with these values.